applications/osb-portal/src/components/MainDrawer/index.tsx

```diff
@@ -0,0 +1,264 @@
+import * as React from "react";
```

Comment:
As already discussed, it would be better the file to be directly named MainDrawer.tsx to simplify search and development